util/topsql/stmtstats/kvexeccount_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stmtstats
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tikv/client-go/v2/tikvrpc"
+)
+
+func TestKvExecCounter(t *testing.T) {
+	manager = newStatementStatsManagerManager()
```

Comment:
What is `newStatementStatsManagerManager` 😅

Reply:
As described by `CreateStatementStats`, if we do not have a back-end `statementStatsManager` yet, `CreateStatementStats` will return nil.
```
// CreateStatementStats try to create and register an StatementStats.
// If we are in the initialization phase and have not yet called SetupTopSQL
// to initialize the top-sql, nothing will happen, and we will get nil. But
// this scene should never appear, because we always call SetupTopSQL before
// starting the server, at this moment we cannot receive connections and will
// not create a valid session. So this case will never happen: "This function
// returns nil, so this valid session will count nothing".
func CreateStatementStats() *StatementStats
```

Reply:
But what's the manager of the manager?

Reply:
Sorry for this typo 🤣

Reply:
Oh. As a typo, not a bad news..😅